### ISSUE TITLE
Language is complex and hard to understand

### ISSUE THREAD

**Issue by vavroom** [w3c/coga#295]
Looking at https://www.w3.org/TR/coga-usable/ I couldn't help but find it rather ironic that the language is quite dense. 

The following is a screenshot of part of the text copy/pasted and analysed on the free Hemingway App website.
![Hemingway App results showing readibility grade 9, good. But 17 phrases with simpler alternatives, 15 of 184 sentences that are hard to read, and 36 of 184 sentences that are very hard to read.](https://user-images.githubusercontent.com/29165258/116946580-58e86e00-ac2f-11eb-9e75-4d3bd3636f51.png)

The text to discuss cognitive disabilities accessibility MUST be easier to read. I know a lot of work has been done to improve the ease of reading and understanding for other documents. I have no doubt some efforts have been made here as well. But it's seriously not a good look to have language that's this dense in this particular context.

The grade reading level is good at 9. But the number of hard and very hard sentences is significant. Simplifying these sentences would greatly improve the text.

**Comment by jpascalides** [comment]
Thank you for your comments. We intended to review this issue before we publish an update.

**Comment by vavroom** [comment]
Thanks for that @jpascalides 

Is there an idea of timeline on when a review and edit of the document is going to happen?

**Comment by jpascalides** [comment]
Hi @vavroom,

The task force is targeting one year out for the next version of the document, but I think that in all likelihood it may be two years before we release an update.

**Comment by vavroom** [comment]
@jpascalides That is... completely not acceptable. That is like every single company we reach out to about accessibility issues that give lip service "thanks for your feedback, we'll get to it, maybe sometime in the future, far far away".

Basically you're saying that you don't care about the negative impact your document is having on cognitively disabled people. The very people you are aiming to help. 

I understand that people from EOWG were providing strong advice and guidance against publishing this document as is without taking more care to make it easier to read. It appears this advice was disregarded. 

One of the most often heard complain I hear when I provide accessibility training to clients (large and small) is that the standards and accompanying documents are hard to understand. This is, I guess, just one more of those things.

Bravo!

**Comment by shawnthompson** [comment]
I agree with @vavroom, a document on cognitive should be easy to read for everyone.

**Comment by brothercake** [comment]
Documentation on accessibility doesn’t need to be accessible, particularly not to the user groups it’s discussing, obvs /s

It’s the “abled man’s burden”.

**Comment by michaelAusbun** [comment]
I recognize and understand the complex nature of writing and publishing notes, technical recommendations, specifications and the like via the W3C process. I also recognize this peer-review process takes time and effort, with the multitude of necessary dependencies and inputs. I have concerns with the timeframe, though.

Access to documents which directly speak to our existence as disabled people should not exclude our understanding or participation. If the process excludes us, then the process is broken. Same goes for documentation regarding us. The old adage, “nothing about us without us.” I fully respect the energy of the volunteers or members who wrote such a needed resource; their energy and care is evident. 

That is why we must do better though. 

@vavroom's point is incredibly valid. If the issue is not enough time for members, I wish to help to alleviate this (though plain language is not my strength). If it is something else, I wish to better understand why the timeline is inordinately lengthy. This is too important to sit on; if one of us does not have access, none of us can have equal access. 

I am willing to help in any way I can.

**Comment by alastc** [comment]
Hi @vavroom, I checked in with the COGA task force and we obviously all agree that making the content as easy to understanding as possible is very important.
 
There have been several plain-English passes over the content already, and once you get to the guidance material it is more straightforward. If you take a snapshot from further down:
 
<img src="https://user-images.githubusercontent.com/216630/120006163-85926a00-bfd0-11eb-92b1-1fffe01f1e0d.png" alt="Screenshot from hemingway app showing section 4.2.2.3 of the document with only 4 sentences of 43 that are considered hard to read, grade 5.">
 
Of course everything can be improved, but it would not be fair to say that this document has not had that attention.
 
Consider that simply specifying someone with cognitive and learning disabilities makes a sentence “very hard to read” according to the Hemingway app. If you take a sentence it considers easy: 
> “Many users cannot learn and remember new design metaphors.”
 
Then specify the users:
> “Many users with cognitive and learning disabilities cannot learn and remember new design metaphors.”
 
Now that sentence is considered “very hard to read”. In most of the document we can omit that, but not in the introduction.
 
It is also worth bearing in mind the whole context:

- There is a plain English summary above where you took that snapshot;
- We are working on an alternative format version (in the WAI website style) which should make the whole thing much easier to browse and digest. That is partly why there is a long timeline on updating the “TR” version, because there is more benefit from creating the WAI-style version.
 
If people do wish to take a pass as the introduction, please do. However, it does need to include quite a lot of context and terminology. It may also be hidden by default in the version being worked on, so it may not be the best use of time.

**Comment by ehind67** [comment]
Thank you for pointing out the language issue out.  There are significant and ongoing efforts to ensure that the document is simple and easy to understand.”

**Comment by joelanman** [comment]
could this be kept open until it is fixed?
